README.md

```diff
@@ -27,6 +27,16 @@ npm start
 
 Now open [localhost:8080](http://localhost:8080)
 
+## Offline First
+
+This tutorial implments ServiceWorker and can be run offline using a Chrome or Firefox browser. To run offline, turn off the server (Ctrl + C) and refresh [localhost:8080](http://localhost:8080) using Chrome or Firefox. The tutorial page will still load.
+
+To remove the offline tutorial from [localhost:8080](http://localhost:8080) visit chrome://serviceworker-internals/ if using Chrome or visit about:serviceworkers if using Firefox and unregister the ServiceWorker. 
```

Comment:
can you check out what it is for Microsoft’s Edge browser? Maybe @jameswestnz can help?